test/fixtures/errors/assert_throws_stack.snapshot

```diff
@@ -0,0 +1,36 @@
+node:assert:*
+      throw err;
+      ^
+
+AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:
++ actual - expected
+
++ Comparison {}
+- Comparison {
+-   bar: true
+- }
+    at Object.<anonymous> (*assert_throws_stack.js:*:*)
+    at Module._compile (node:internal*modules*cjs*loader:*:*)
+    at Module._extensions..js (node:internal*modules*cjs*loader:*:*)
+    at Module.load (node:internal*modules*cjs*loader:*:*)
+    at Module._load (node:internal*modules*cjs*loader:*:*)
+    at Function.executeUserEntryPoint [as runMain] (node:internal*modules*run_main:*:*)
+    at node:internal*main*run_main_module:*:* {
```

Comment:
We need to get all of this out of the snapshot, so that changes to the CommonJS loader don’t break tests such as this one. You can use `Error.stackTraceLimit = 1` or similar for this (see the similar tests nearby).